getting-started/installation/general-installation/installation-community-containerized.markdown

```diff
@@ -0,0 +1,238 @@
+    [ YES ] Check if agent is acting as a policy server: Acting as a policy server
+    [ YES ] Check private key: OK at '/var/cfengine/ppkeys/localhost.priv'
+    [ YES ] Check public key: OK at '/var/cfengine/ppkeys/localhost.pub'
+    [ NO  ] Check persistent classes DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/cf_state.lmdb'
+    [ NO  ] Check checksums DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/checksum_digests.lmdb'
+    [ NO  ] Check observations DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/cf_observations.lmdb'
+    [ NO  ] Check file stats DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/stats.lmdb'
+    [ NO  ] Check locks DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/cf_lock.lmdb'
+    [ NO  ] Check performance DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/performance.lmdb'
+    [ NO  ] Check lastseen DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/cf_lastseen.lmdb'
+    ```
+    ```
+    $ docker exec -it cfengine-demo-cfengine-host-1 bash -c "/usr/local/sbin/cf-agent --self-diagnostics"
+    self-diagnostics for agent using workdir '/var/cfengine'
+    self-diagnostics for agent using inputdir '/var/cfengine/inputs'
+    self-diagnostics for agent using logdir '/var/cfengine'
+    self-diagnostics for agent using statedir '/var/cfengine/state'
+    [ YES ] Check that agent is bootstrapped: 192.168.16.2
+    [ NO  ] Check if agent is acting as a policy server: Not acting as a policy server
+    [ YES ] Check private key: OK at '/var/cfengine/ppkeys/localhost.priv'
+    [ YES ] Check public key: OK at '/var/cfengine/ppkeys/localhost.pub'
+    [ NO  ] Check persistent classes DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/cf_state.lmdb'
+    [ NO  ] Check checksums DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/checksum_digests.lmdb'
+    [ NO  ] Check observations DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/cf_observations.lmdb'
+    [ NO  ] Check file stats DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/stats.lmdb'
+    [ NO  ] Check locks DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/cf_lock.lmdb'
+    [ NO  ] Check performance DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/performance.lmdb'
+    [ NO  ] Check lastseen DB: Unable to diagnose LMDB file (not implemented) for '/var/cfengine/state/cf_lastseen.lmdb'
+    ```
+5. Stop services and cleanup
+    ```
+    $ docker compose -f compose.yaml down --remove-orphans --rmi "local" -v
+    ```
+
+## Supporting notes and references
+### Enabling systemd for WSL
+Follow the instructions provided here: [Use systemd to manage Linux services with WSL](https://learn.microsoft.com/en-us/windows/wsl/systemd).
```

Comment:
This seems off topic and can be removed.

Reply:
This was added as WSL used to run docker doesn't enable systemd by default.
Removed this as suggested.